docs/source/overview/delivery_services.rst

```diff
@@ -505,6 +505,9 @@ Max Origin Connections
 ----------------------
 The maximum number of TCP connections individual :term:`Mid-tier cache servers` are allowed to make to the `Origin Server Base URL`. A value of ``0`` in this field indicates that there is no maximum.
 
+
+		.. note:: Max Origin Connections can be made per-:ref:`Cache Group <cache-groups>` by setting the :ref:`ds-regional` field.
```

Comment:
indentation here is causing this note to be rendered as a block quote - it's not a quote, so it shouldn't be indented.

Reply:
Unindented in 1640dc9ec3